tests/unit/cred-revocation.test.js

```diff
@@ -1,31 +1,34 @@
-import { isRevocationCheckNeeded } from '../../src/utils/vc';
+import {
+  isRevocationCheckNeeded,
+  expandedStatusProperty
+} from '../../src/utils/vc';
 
 describe('Check isRevocationCheckNeeded', () => {
   test('isRevocationCheckNeeded returns true when credentialStatus is present and forceRevocationCheck is true and revocation API is not given', () => {
-    expect(isRevocationCheckNeeded({}, true, undefined)).toBe(true);
+    expect(isRevocationCheckNeeded({ [`${expandedStatusProperty}`]: {} }, true, undefined)).toBe(true);
```

Comment:
Curious, why the string interpolation?

```
expandedStatusProperty === `${expandedStatusProperty}`
```
right?

Reply:
was using it for more than that at first, have updated